models/user.rb

```diff
@@ -11,16 +11,24 @@ def update_profile_image(icon)
     twitter.update_profile_image(icon)
   end
 
+  def tweet_changing_icon(icon_url)
+    twitter.update <<TWEET
```

Comment:
you can use [squiggly_heredoc](http://docs.ruby-lang.org/ja/2.3.0/doc/spec=2fliteral.html#here)(`<<~HEREDOC`), because yryr-icon works on ruby 2.3.0 💎  https://github.com/flada-auxv/yryr-icon/blob/master/Gemfile#L3